### PR TITLE
feat: Add QR code button to the user profile  - WPB-10540

### PR DESCRIPTION
**Pull request by KaterinaWire**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-10540" title="WPB-10540" target="_blank">StoryWPB-10540</a>  [iOS] Your Profile page
  </td></table>
  <br />
 

### Issue

Add the QR code button to self user profile:

Screenshot 2024-10-10 at 07 42 28

### Checklist

- [x] Title contains a reference JIRA issue number like `[WPB-XXX]`.
- [x] Description is filled and free of optional paragraphs.
- [ ] Adds/updates automated tests.

---

### UI accessibility checklist

_If your PR includes UI changes, please utilize this checklist:_
- [ ] Make sure you use the API for UI elements that support large fonts.
- [ ] All colors are taken from WireDesign.ColorTheme or constructed using WireDesign.BaseColorPalette.
- [ ] New UI elements have Accessibility strings for VoiceOver.

### CODE REVIEW

File: wire-ios/Wire-iOS/Sources/UserInterface/SelfProfile/ProfileHeaderViewController.swift [hunk 1]
```diff
@@ -355,6 +380,12 @@ final class ProfileHeaderViewController: UIViewController {
         }
     }
 
+    private func updateQRCodeButton() {
+        let qrCodeAction = UIAction { _ in }
```
Comment (jullianm) [inline wire-ios/Wire-iOS/Sources/UserInterface/SelfProfile/ProfileHeaderViewController.swift:384]: We're not doing anything here but I assume this is part of another ticket.

**Review by jullianm — APPROVED** [review]
LGTM

File: wire-ios/Wire-iOS/Sources/UserInterface/SelfProfile/ProfileHeaderViewController.swift (hunk 1 above)
Comment (KaterinaWire) [reply wire-ios/Wire-iOS/Sources/UserInterface/SelfProfile/ProfileHeaderViewController.swift:384]: It's here https://github.com/wireapp/wire-ios/pull/1957

**Review by agisilaos — APPROVED** [review]

**Review by agisilaos — COMMENTED** [review]
Looking good! One thing regarding snapshots. We're trying to reduce the size of them by using a tool called [ImageOptim](https://imageoptim.com/mac) Please use the tool to reduce the size of the images.

**Comment by KaterinaWire** [comment]
> Looking good! One thing regarding snapshots. We're trying to reduce the size of them by using a tool called [ImageOptim](https://imageoptim.com/mac) Please use the tool to reduce the size of the images.

Done

**Review by jullianm — APPROVED** [review]

**Review by agisilaos — APPROVED** [review]

**Pull request merged**